### DISCUSSION TITLE
How to implement log_not_found off

### DISCUSSION THREAD

**Discussion by Horus-K** [apache/apisix#7015] [Q&A]
I want to implement the configuration of nginx

```
location = /favicon.ico {
      log_not_found off;
      access_log off;
}
```
what should i do

**Comment by tzssangglass** [comment]
see: https://github.com/apache/apisix/blob/master/docs/en/latest/customize-nginx-configuration.md

**Comment by Horus-K** [comment]
Can it be implemented at the route level?

**Reply by tzssangglass** [reply]
unsupported
